idea-plugin/build.gradle.kts

```diff
@@ -62,7 +59,8 @@ compose.resources {
 }
 
 composeCompiler {
-    stabilityConfigurationFile = rootProject.layout.projectDirectory.file("stability_config.conf")
+    enableStrongSkippingMode = true
+    stabilityConfigurationFile = layout.projectDirectory.file("stability_config.conf")
```

Comment:
Should we share this config and enable strong-skip for App and Plugin?

Reply:
19053bbde6b0ec2186596cbf7ec1177bb62bb92f

Reply:
will be removed after release kotlin 2.0.20 
https://kotlinlang.org/docs/whatsnew-eap.html#strongskipping-mode-enabled-by-default

Reply:
Nice, will remove it then.